doc/conf.py

```diff
@@ -1,59 +1,23 @@
-# -- Path setup --------------------------------------------------------------
+from urllib.request import urlopen
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+_conf_url = \
+        "https://raw.githubusercontent.com/inducer/sphinxconfig/main/sphinxconfig.py"
+with urlopen(_conf_url) as _inf:
+    exec(compile(_inf.read(), _conf_url, "exec"), globals())
 
-
-# -- Project information -----------------------------------------------------
-
-project = "arraycontext"
 copyright = "2021, University of Illinois Board of Trustees"
+author = "Arraycontext Contributors"
 
 ver_dic = {}
-exec(
-        compile(
-            open("../arraycontext/version.py").read(),
-            "../arraycontext/version.py", "exec"),
-        ver_dic)
+exec(compile(open("../arraycontext/version.py").read(), "../arraycontext/version.py",
+    "exec"), ver_dic)
 version = ".".join(str(x) for x in ver_dic["VERSION"])
-# The full version, including alpha/beta/rc tags.
 release = ver_dic["VERSION_TEXT"]
 
-# -- General configuration ---------------------------------------------------
-
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = [
-    "sphinx.ext.autodoc",
-    "sphinx.ext.doctest",
-    "sphinx.ext.intersphinx",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.graphviz",
-    "sphinx_copybutton",
+extensions += [
+    "sphinx_autodoc_typehints",
```

Comment:
Is this required? I was under the impression the typehints functionality is in core sphinx these days?

Reply:
Yeah, it seems to work fine without the extension. Removed.